scipy/optimize/tests/test_direct.py

```diff
@@ -214,3 +239,80 @@ def test_inf_fun(self, locally_biased):
         result = direct(self.inf_fun, bounds,
                         locally_biased=locally_biased)
         assert result is not None
+
+    @pytest.mark.parametrize("len_tol", [-1, 2])
+    def test_len_tol_validation(self, len_tol):
+        error_msg = "len_tol must be between 0 and 1."
+        with pytest.raises(ValueError, match=error_msg):
+            direct(self.styblinski_tang, self.bounds_stylinski_tang,
+                   len_tol=len_tol)
+
+    @pytest.mark.parametrize("vol_tol", [-1, 2])
+    def test_vol_tol_validation(self, vol_tol):
+        error_msg = "vol_tol must be between 0 and 1."
+        with pytest.raises(ValueError, match=error_msg):
+            direct(self.styblinski_tang, self.bounds_stylinski_tang,
+                   vol_tol=vol_tol)
+
+    @pytest.mark.parametrize("f_min_rtol", [-1, 2])
+    def test_fmin_rtol_validation(self, f_min_rtol):
```

Comment:
I noticed that this condition is documented for `vol_tol` and `len_tol` but not `fmin_rtol`

Reply:
Updated the docstring, good catch.